locals.tf

```diff
@@ -51,8 +65,8 @@ locals {
     autoscaling         = false
     placement_group_id  = null
     container_runtime   = "docker"
-    wait_for_pool_ready = false
     tags                = []
+    wait_for_pool_ready = true
```

Comment:
The provider default is `False` we can set it to true un tkap if needed for addons